guides/partials.md

```diff
@@ -1,36 +1,54 @@
 ---
-title: DustJS | Getting Started
+title: DustJS | Partials
 layout: guides
 permalink: /guides/partials/
 ---
 
-## Partials
+# Partials
 
-A Dust partial promotes "DRY" behavior for reusable/shared markup. Partials themselves can be composed of one or more partials. Partials rely on the JSON/context of the parent template invoking it. Lets peek under the covers to see how a Dust partial works.
+A Dust partial promotes *DRY* ("don't repeat yourself") conventions in shared markup. Breaking templates up into partials can make them easier to maintain and reuse, and partials themselves can be composed of one or more partials. This is the basis for components or reusable templates for tasks like including a common header and footer on multiple pages. Lets peek under the covers to see how a Dust partial works.
 
-You can have multiple .dust files and reference one Dust template as part of another one. This is the basis for components or reusable templates for tasks like a common header and footer on multiple pages. Note that the .dust file extension is used here in examples, but .tl is also commonly seen. Since it only matters to the build process you can use whatever extension works for you.
+A partial template is added to another template using Dust's partial syntax. Note that the tag is self-closing (`/}`).
```

Comment:
I think the "(`/}`)" example may be more confusing than helpful. Perhaps we could just link to the partials syntax: "/docs/syntax/#partial"